src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -175,6 +175,11 @@ public BrowserTest(int secondsBeforeTimeout) {
         ensureActiveTabIsNotClosed();
     }
 
+    public BrowserTest(String appium){
+
```

Comment:
Setting seconds before timeout, and ensuring the active tab is not closed does not work for Appium?
I believe that it would probably be better to ensure those methods don't do anything for an Android App test, instead of having the extra constructor